rb/lib/selenium/webdriver/remote/bridge.rb

```diff
@@ -42,6 +42,7 @@ def initialize(url:, http_client: nil)
           @http = http_client || Http::Default.new
           @http.server_url = uri
           @file_detector = nil
+          @authenticator_id = nil
```

Comment:
The bridge is supposed to be mostly stateless, so what if instead, we change an interface to return the virtual authenticator when it's added and allow to remove it later as a parameter? Something like this:

```ruby
authenticator = driver.add_virtual_authenticator(options)
driver. remove_virtual_authenticator(authenticator)
```

This way we can avoid storing authenticator ID internally. It would also allow us to have multiple virtual authenticators.

Reply:
I cleared bridge of the authenticator methods and created a separate class `VirtualAuthenticator` as you suggested. Now we can get and remove the authenticator object without storing its id in bridge.